1-installation/index.md

```diff
@@ -136,9 +136,17 @@ To make sure we're all using the same tools, let's go ahead and install <a href=
 
 ## Step 5: Install GitHub
 
-This app will make it easier to publish your website online at the end of the day.
+This app will make it easier to publish your website online at the end of the day, or any other time, for that matter.
 
-// TODO: install links and instructions
```

Comment:
ok... I'll leave it as it is, since we'll be wrapping it in a hint div/something anyway, so this PR probably will not be included in this form...
just leaving it open, so we don't forget